keps/sig-node/2400-node-swap/README.md

```diff
@@ -290,6 +361,10 @@ nodes could improve better resource pressure handling and recovery.
 
 This user story is addressed by scenario 1 and 2, and could benefit from 3.
 
+Note: critical / high-priority pods would not be able to access swap, but can
+still be configured otherwise to gain swap access. In the future, APIs would
```

Comment:
APIs or additional kubelet swap modes?

Reply:
I wouldn't be surprised if we punt the API driven swap forever, and would first just introduce a critical pod limited swap mode or something.

Reply:
These are good questions that we don't have the answers to yet :)
I do think that swap will probably be more customizable, but I agree that pods will likely never be able to request it directly. However at this stage I'm not entirely sure if only swap behavior modes will be added or we will add more APIs.

In any case, I've clarified that adding only more swap behaviors (not APIs) is possible